hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/InitializeEntityBalanceMigration.java

```diff
@@ -87,4 +89,12 @@ protected void doMigrate() {
         int count = jdbcOperations.update(INITIALIZE_ENTITY_BALANCE_SQL);
         log.info("Initialized {} entities balance in {}", count, stopwatch);
     }
+
+    @EventListener
+    public void reRunMigration(InitializeBalanceEvent event) {
+        log.info(
+                "Re-running the InitializeEntityBalanceMigration on InitializeBalanceEvent created at {}",
+                event.getTimestamp());
```

Comment:
don't see much value logging the event's created timestamp
```suggestion
        log.info("Re-running the InitializeEntityBalanceMigration on InitializeBalanceEvent");
```